lib/units/storage/amazons3.js

```diff
@@ -0,0 +1,197 @@
+var http = require('http')
+var util = require('util')
+var path = require('path')
+
+var express = require('express')
+var validator = require('express-validator')
+var bodyParser = require('body-parser')
+var formidable = require('formidable')
+var Promise = require('bluebird')
+var uuid = require('node-uuid')
+var fs = require('fs')
```

Comment:
Move `fs` to the first group. The idea is that built-in modules go to the first group. Then all 3rd-party modules go to the 2nd group. And then finally all local requires to into the 3rd group.